src/askContainer/commands/viewAllSkills.ts

```diff
@@ -206,16 +199,33 @@ export class ViewAllSkillsCommand extends AbstractCommand<void> {
 
         allSkillsQP.onDidAccept(async() => {
             if (allSkillsQP.activeItems.length !== 0) {
-                allSkillsQP.ignoreFocusOut = false;
-                this.checkSkillExist(context, allSkillsQP)
+                const skillInfo = this.skillInfoMap.get(allSkillsQP.activeItems[0].label)!;
+                this.checkSkillExist(context, skillInfo)
                     .then(() => {
                         const executeCommand = 'askContainer.skillsConsole.cloneSkill';
                         void vscode.commands.executeCommand(
                             executeCommand,
-                            this.skillInfoMap.get(allSkillsQP.activeItems[0].label)
+                            skillInfo
                         );
                     })
-                    .then(undefined, e => {})
+                    .then(undefined, async (error) => {
+                        if (error.statusCode === 404) {
+                            allSkillsQP.ignoreFocusOut = true;
```

Comment:
Set `ignoreFocusOut = true` to avoid closing the quickPick UI when user click the `OK` button.